src/kprofile.cpp

```diff
@@ -5,6 +5,27 @@
 #include <QMessageBox>
 #include <QWebEngineSettings>
 
+QString askForSaveFilePath(const QString& suggestedName)
+{
+    const auto app = KiwixApp::instance();
+    const QString suggestedPath = app->getPrevSaveDir() + "/" + suggestedName;
+    const QString extension = suggestedName.section('.', -1);
+    const QString filter = extension != '.' ? "(*" + extension + ")" : "";
```

Comment:
In which case can `extension` be `"."`? (BTW, it's better to use double than single quotes even though Qt works with the latter too).

Reply:
I have looked through the Qt Doc, and "." cannot appear from section, as it excludes the seperator. I simply checked whether the retrieved extension is empty.